__tests__/app.test.js

```diff
@@ -123,3 +123,41 @@ describe('GET /api/articles', () => {
             });
     });
 });
+describe('GET /api/articles/:article_id/comments', () => {
+    test('should return a 200 status code', () => {
+        return request(app).get('/api/articles/3/comments').expect(200);
+    })
+    test('should respond with an array of comment objects given the article_id', () => {
+        return request(app)
+            .get('/api/articles/3/comments')
+            .then(({ body }) => {
+                expect(body.comments.length).toBeGreaterThan(0);
+                body.comments.forEach((comment) => {
+                    expect.objectContaining({
+                        comment_id: expect.any(Number),
+                        votes: expect.any(Number),
+                        created_at: expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        article_id: expect.any(Number),
+                    }) 
+                });
+            });
+    })
+    test('should return a 400 error if no article_id', () => {
```

Comment:
what about if the article id is valid but non-existent? We need another test for this too